tests/cli/test_check.py

```diff
@@ -0,0 +1,184 @@
+      |   |   |   | driving  | string |         | vairavimas  |         | open   | protected
+                             | enum   |         | l           | 'left'  | open   | public
+                             |        |         | r           | 'right' | open   | package
+                             |        |         |             |         |        |
+      |   |   | City         |        | name    | miestas     |         |        |
+      |   |   |   | name     | string |         | pavadinimas |         | open   | private
+      |   |   |   | country  | ref    | Country | salis       |         | open   | private
+    '''))
+
+    cli.invoke(rc, [
+        'check',
+        tmp_path / 'manifest.csv',
+    ])
+
+
+def test_check_eli(context: Context, rc, cli: SpintaCliRunner, tmp_path):
+    create_tabular_manifest(context, tmp_path / 'manifest.csv', striptable('''
+    d | r | b | m | property | type   | ref     | source      | prepare | access | eli
+    datasets/gov/example     |        |         |             |         |        |
+      | data                 | sql    |         |             |         |        |
+                             |        |         |             |         |        |
+      |   |   | Country      |        | code    | salis       |         |        | https://example.com/law/1
+      |   |   |   | code     | string |         | kodas       |         | public | https://example.com/law/2
+      |   |   |   | name     | string |         | pavadinimas |         | open   |
+      |   |   |   | driving  | string |         | vairavimas  |         | open   | https://example.com/law/3
+                             | enum   |         | l           | 'left'  | open   | https://example.com/law/4
+                             |        |         | r           | 'right' | open   | https://example.com/law/5
+    '''))
+
+    cli.invoke(rc, [
+        'check',
+        tmp_path / 'manifest.csv',
+    ])
+
+
+def test_check_count(context: Context, rc, cli: SpintaCliRunner, tmp_path):
+    create_tabular_manifest(context, tmp_path / 'manifest.csv', striptable('''
+    d | r | b | m | property | type   | ref     | source      | prepare | access | count
+    datasets/gov/example     |        |         |             |         |        |
+      | data                 | sql    |         |             |         |        | 4
+                             |        |         |             |         |        |
+      |   |   | Country      |        | code    | salis       |         |        | 5 
+      |   |   |   | code     | string |         | kodas       |         | public | 6
+      |   |   |   | name     | string |         | pavadinimas |         | open   |
+      |   |   |   | driving  | string |         | vairavimas  |         | open   | 7
+                             | enum   |         | l           | 'left'  | open   | 8
+                             |        |         | r           | 'right' | open   | 9
```

Comment:
Pagal ką atliekamas šis skaičiavimas (`count` stulpelis) ? Ar čia kol kas tiesiog testiniai duomenis, kurie turės būti keičiami implementavus `count` logiką?

Reply:
dabar tiesiog testiniai duomenys, jie nėra skaičiuojami. Jie tiesiog kopijuojami arba tikrinamas formato validumas (būtent `spinta check` tikrina validumą DSA, kuris gali būti ir automatiškai sugeneruotas, ir užpildytas arba papildytas rankiniu būdu. Tai čia tiesiog tikrinama jau užpildyto DSA struktūra, ar ji teisinga